packages/e2e-test-app/NuGet.Config

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think this file is in the wrong folder. By placing it here, either nuget restore is not seeing this file at all, or nuget restore is placing the nuget packages into the `packages/e2e-test-app/packages` folder when the Folly project is looking at `packages/e2e-test-app/windows/packages`. Move this into the folder alongside the solution file.